aiida/orm/utils/loaders.py

```diff
@@ -8,7 +8,7 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 """Module with `OrmEntityLoader` and its sub classes that simplify loading entities through their identifiers."""
-from abc import abstractclassmethod
+from abc import abstractmethod
```

Comment:
same as `abstractproperty` what is wrong with this?

Reply:
again deprecated: https://docs.python.org/3/library/abc.html?highlight=abstractproperty#abc.abstractclassmethod